test/crash_test/src/crash_mgmt.c

```diff
@@ -42,16 +43,25 @@ struct mgmt_group crash_test_nmgr_group = {
     .mg_group_id = MGMT_GROUP_ID_CRASH
 };
 
+static char how_str[64];
```

Comment:
The size of `how_str` array can be much smaller since the requested crash methods are very small in length. Let's reduce that.